src/WebDriver.php

```diff
@@ -575,7 +575,7 @@ public function getValue($xpath)
     {
         $element = $this->findElement($xpath);
         $elementName = strtolower($element->getTagName());
-        $elementType = strtolower($element->getAttribute('type'));
+        $elementType = strtolower($element->getAttribute('type') ?? '');
```

Comment:
Let's add fallback to "text" and please search for other places (line 650)

Reply:
better to update "\OAndreyev\Mink\Driver\WebDriver::getAttribute":
if $element->getTagName() === input and $element->getAttribute('type') === null, return text as default and fallback